### ISSUE TITLE
Refresh preview on option change in Section or Column

### ISSUE THREAD

**Issue by zecob7** [elementor/elementor#11224]
Hello,

I added a SWITCHER option in Section and Column both. The option is working perfectly on frontend view. The option is background JS effect.

But the preview is not showing any effect becuase the preview is not refreshed. Here is my code you can try:

`

/**
 *  Add options in existing elements or sections
 */
add_action( 'elementor/element/after_section_start', function( $element, $section_id, $args ) {

	// Section (Row) Options
	if ( 'section' === $element->get_name()  ) {
			
		$element->add_control(
			'myprefix-animated-bg-effect',
			[
				'label'			=> esc_attr__( 'Add animated bg effect in this Section?', 'myprefix' ),
				'type'			=> Elementor\Controls_Manager::SWITCHER,
				'label_on'		=> esc_attr__( 'Yes', 'myprefix' ),
				'label_off'		=> esc_attr__( 'No', 'myprefix' ),
				'return_value'	=> 'yes',
				'default'		=> '',
			]
		);

	}

	// Column options
	if ( 'column' === $element->get_name()  ) {
	
		$element->add_control(
			'myprefix-animated-bg-effect',
			[
				'label'			=> esc_attr__( 'Add animated bg effect in this Column?', 'myprefix' ),
				'type'			=> Elementor\Controls_Manager::SWITCHER,
				'label_on'		=> esc_attr__( 'Yes', 'myprefix' ),
				'label_off'		=> esc_attr__( 'No', 'myprefix' ),
				'return_value'	=> 'yes',
				'default'		=> '',
			]
		);

	}
	
}, 10, 3 );

`
Basically what I want is to refresh the preview when I change thw SWITHCER option.

**Comment by zecob7** [comment]
Any update on this?? Is this feature available or need to report as bug or feature??
